packages/fortinet_fortigate/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -225,7 +225,56 @@ processors:
       field: fortinet.firewall.msg
       target_field: message
       ignore_missing: true
-
+  - rename:
+      field: fortinet.firewall.dstmac
+      target_field: destination.mac
+      ignore_missing: true
+  - gsub:
+      field: destination.mac
+      pattern: "\\:"
+      replacement: "\\-"
```

```suggestion
      pattern: '\:'
      replacement: '\-'
```
Is the escape actually needed? Neither of `:` or `-` are regex-meaningful in this context. ISTM that this could just be
```suggestion
      pattern: ':'
      replacement: '-'
```